src/components/CategoryListContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import { selectCategory } from '../redux_module/selectedSlice';
+import ButtonList from './ButtonList';
+
+export default function CategoryListContainer() {
+  const dispatch = useDispatch();
+
+  const names = useSelector((state) => state.groups.categories);
+  const selected = useSelector((state) => state.selected.category);
+  const handleClick = (value) => dispatch(selectCategory(value));
```

Comment:
저는 의미상으로 구분되는 경우 빈줄을 넣어서 구분해주는 편이에요.
리덕스를 불러오는 부분과 이벤트 핸들 함수 이렇게 구분지어 주면 어떨까요?

```suggestion
  const names = useSelector((state) => state.groups.categories);
  const selected = useSelector((state) => state.selected.category);
  
  const handleClick = (value) => dispatch(selectCategory(value));
```

Reply:
좋습니다! 다른 곳에서도 적용해줬습니다.